ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -161,6 +167,17 @@ const TransactionsScreen: React.FC<Props> = (props: Props) => {
       <Label color={"red"}>{I18n.t("cardComponent.removeCta")}</Label>
     </ButtonDefaultOpacity>
   );
+
+  if (props.isLoadingDelete) {
+    return (
+      <LoadingErrorComponent
+        isLoading={props.isLoadingDelete}
+        loadingCaption={loadingCaption}
+        onRetry={onRetry}
+      />
+    );
+  }
+
   return (
```

Comment:
Can this reduce your code size?
```suggestion
  return props.isLoadingDelete ? (
    <LoadingErrorComponent
      isLoading={props.isLoadingDelete}
      loadingCaption={I18n.t("cardComponent.deleteLoading")}
      onRetry={onRetry}
    />
  ) : (
```